Source/SharpNeedle/Glitter/Effect.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace SharpNeedle.Glitter;
+
+using SharpNeedle.BINA;
+
+[NeedleResource("glitter/effect", @"\.(gle|effect)$")]
+public class Effect : BinaryResource
+{
+    public new static readonly uint Signature = BinaryHelper.MakeSignature<uint>("FIRG");
+    public uint Version { get; set; } = 0x01060000;
+    public LinkedList<EffectParameter> Parameters { get; set; } = new();
+    
+    public override void Read(BinaryObjectReader reader)
+    {
+        reader.EnsureSignature(Signature);
+
+        Version = reader.Read<uint>();
```

Comment:
Since this doesn't support anything other than GRIF 1.6 yet. Just check for the version and throw a `NotSupportedException` for now. Until older versions are figured out. So we can merge this.

Reply:
Resolved in [3dda7dc](https://github.com/hedge-dev/SharpNeedle/pull/8/commits/3dda7dc61e5525a10f9571d45d260d9fa7034ebb).